aws/apigateway/enable-access-logging/metadata.rego

```diff
@@ -0,0 +1,17 @@
+package cloud-metadata.aws.apigateway.enable-access-logging
+
+__rego_metadata__ := {
+	"id": "AWS061",
```

Comment:
Is this the format for the ID? Will it continue the pattern of provider then a incremental number or should be be using something more generic like `AVD00001` sort of thing?

The AquaID can easily be retrofitted back to tfsec when we know what for it will take

Reply:
I think having an AVD-based ID makes sense for the centralized metadata. Each product/project that leverages it is free to **also** use its own ID, but **must** at least use the central AVD ID. This gives us:

1. the ability to correlate issues across products/projects today
2. the ability to correlate issues across future projects without having to hardcode other project fields into the top-level metadata